src/standalone/executionAnalysis/symbols.ts

```diff
@@ -259,8 +259,10 @@ export class NotebookDocumentSymbolTracker {
                     });
 
                     e.cellChanges.forEach((change) => {
-                        this._requestCellSymbols(change.cell, false).then(noop, noop);
-                        this._updateCellStatus(change.cell, CellExecutionStatus.Stale);
+                        if (change.document) {
```

Comment:
I think this should be done only when there are changes to the text in the cells